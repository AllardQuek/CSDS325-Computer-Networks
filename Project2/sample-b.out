REQ: GET / HTTP/1.0
REQ: Host: www.icir.org
REQ: User-Agent: CWRU EECS 325 Client 1.0
Command: ./proj2 -c -u http://www.icir.org -o mallman.html
