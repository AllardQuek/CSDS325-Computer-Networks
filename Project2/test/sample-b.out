REQ: GET / HTTP/1.0
REQ: Host: www.icir.org
REQ: User-Agent: CWRU CSDS 325 Client 1.0
